### PR TITLE
Scapy 2.4.0rc5-40: Little but obvious code improvements. [PR is ready for merge to the master]

### PR DESCRIPTION
**Pull request by karpierz**
Scapy Version: 2.4.0rc5-40
System: Windows7/Windows10
Python Version: 2.7.14/3.4.4/3.6.4

### CODE REVIEW

**Comment by karpierz** [comment]
PR is ready for review and merge to the master.

File: scapy/asn1fields.py [hunk 1]
```diff
@@ -441,10 +435,7 @@ def __init__(self, name, default, *args, **kwargs):
             raise ASN1_Error(err_msg)
         self.implicit_tag = None
         for kwarg in ["context", "explicit_tag"]:
-            if kwarg in kwargs:
-                setattr(self, kwarg, kwargs[kwarg])
-            else:
-                setattr(self, kwarg, None)
+            setattr(self, kwarg, kwargs.get(kwarg))
```
Comment (gpotter2) [inline scapy/asn1fields.py]: Shouldn’t it be
`kwargs.get(kwarg, None)`
?

Comment (karpierz) [reply scapy/asn1fields.py]: kwargs.get(kwarg) is exactly the same as kwargs.get(kwarg, None)

default is None

But if you think that strict "explicit is better than implicit" is more valuable/accurate
I am quite open for agreement with you in this matter :)

Comment (gpotter2) [reply scapy/asn1fields.py]: You are right. I thought it would have triggered an undefined error, but I was wrong

Comment (p-l-) [reply scapy/asn1fields.py]: @gpotter2 that's the case for `.pop()`... which is a bit weird as you might think both would have the same API.

Comment (karpierz) [reply scapy/asn1fields.py]: Exactly:-)

**Comment by karpierz** [comment]
PR is ready for merge to the master.

**Comment by gpotter2** [comment]
Hi !
I was able to replicate the bug that made you restart this PR,
I'll fix it soon :)

**Comment by karpierz** [comment]
Fine. Thanks :)

**Comment by karpierz** [comment]
PR is ready for merge to the master.

**Pull request merged**